style.css

```diff
@@ -1,3 +1,769 @@
-body {
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap');
+@import url('http://fonts.cdnfonts.com/css/cocogoose');
+
+* {
   box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+:root {
+  --main-color-black: #5c5d5f;
+  --main-color-gray: #9b9595;
+  --main-color-orange: #ec5242;
+}
```

Comment:
- [ ] Kindly it is required match to the template as much as possible that's why the pages should use the exact same colors, for text, backgrounds, borders, etc, from the original design.
- Basic color swatch: #272A31, #EC5242, #D3D3D3

**Hint:**
```css
:root {
  --main-color-black: #272a31;
  --main-color-gray: #d3d3d3;
  --main-color-orange: #ec5242;
}
```